frame_2D_alg/alternative versions/P_blobs.py

```diff
@@ -141,7 +141,7 @@ def output_path(input_path, suffix):
 
     if verbose:
         start_time = time()
-        print("Converting to image to blobs...")
+        print("Converting the image(frame) to blobs...")
```

Comment:
Thanks, this is obsolete version anyway.
Alternative versions are generally obsolete.